tutorials/02_installation.md

```diff
@@ -31,31 +31,52 @@ wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
 
 ```{.sh}
 sudo apt-get update
-sudo apt-get install libignition-physics3-dev
+sudo apt-get install libignition-physics<#>-dev
 ```
 
+Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on which version you need.
+
 **Build from source**
 
-Run the following to install dependencies
-```{.sh}
-sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
-sudo apt-get build-dep -y ignition-physics3
-```
+1. Run the following to install dependencies
 
-Clone the ign-physics repository from GitHub
-```{.sh}
-git clone https://github.com/ignitionrobotics/ign-physics -b master
-```
+    ```{.sh}
+    sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
+    sudo apt-get build-dep -y ignition-physics<#>-dev
+    ```
+    Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on which version you need.
+
+
+2. Use gcc-8:
+
+    ```
+    sudo apt update
+    sudo apt-get -y install g++-8
+    sudo update-alternatives --install \
+      /usr/bin/gcc gcc /usr/bin/gcc-8 800 \
+      --slave /usr/bin/g++ g++ /usr/bin/g++-8 \
+      --slave /usr/bin/gcov gcov /usr/bin/gcov-8
+    ```
+
+3. Clone the repository
+
+    ```
+    git clone https://github.com/ignitionrobotics/ign-physics -b ign-physics<#>-dev
```

Comment:
`ign-physics<#>` is valid for release branches (remove `-dev` suffix), but users may also want the `master` branch